cinema_service/urls.py

```diff
@@ -2,10 +2,17 @@
 from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import path, include
+from drf_spectacular.views import SpectacularAPIView, SpectacularSwaggerView, SpectacularRedocView
+from rest_framework_simplejwt.views import TokenObtainPairView, TokenRefreshView
 
 urlpatterns = [
-    path("admin/", admin.site.urls),
-    path("api/cinema/", include("cinema.urls", namespace="cinema")),
-    path("api/user/", include("user.urls", namespace="user")),
-    path("__debug__/", include("debug_toolbar.urls")),
-] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
+                  path("admin/", admin.site.urls),
+                  path("api/cinema/", include("cinema.urls", namespace="cinema")),
+                  path("api/user/", include("user.urls", namespace="user")),
+                  path("__debug__/", include("debug_toolbar.urls")),
```

Comment:
The debug toolbar is included in the URL patterns. Make sure to disable or restrict access to the debug toolbar in production to prevent exposing sensitive information.